spring-cloud-gcp-pubsub-stream-binder/src/main/java/com/google/cloud/spring/stream/binder/pubsub/properties/PubSubConsumerProperties.java

```diff
@@ -57,4 +59,34 @@ public String getSubscriptionName() {
 	public void setSubscriptionName(String subscriptionName) {
 		this.subscriptionName = subscriptionName;
 	}
+
+	public DeadLetterPolicy getDeadLetterPolicy() {
+		return deadLetterPolicy;
+	}
+
+	public void setDeadLetterPolicy(DeadLetterPolicy deadLetterPolicy) {
```

Comment:
Would it be better to use the [`DeadLetterPolicy` proto object in the client libs](https://github.com/googleapis/java-pubsub/blob/7d5bc463cf62395137b88261e01ceb792d379001/samples/snippets/src/main/java/pubsub/UpdateDeadLetterPolicyExample.java#L61) instead of creating a new custom wrapper?

The proto object is a little more complex, but it would reduce code duplication and also stay up to date if new settings are introduced to dead letter policy.

Reply:
:+1: